dydx/dydxViews/dydxViews/_v4/Portfolio/Components/Sections/dydxPortfolioPositionsView.swift

```diff
@@ -112,23 +119,16 @@ public class dydxPortfolioPositionItemViewModel: PlatformViewModel {
     private func createMain(parentStyle: ThemeStyle) -> some View {
         HStack {
             VStack(alignment: .leading, spacing: 0) {
-                HStack(spacing: 2) {
-                    Text(size ?? "")
-                        .themeFont(fontType: .number, fontSize: .small)
-
-                    token?.createView(parentStyle: parentStyle.themeFont(fontSize: .smallest))
-                }
+                Text(marketValue ?? "--")
+                    .themeFont(fontType: .number, fontSize: .small)
 
                 HStack(spacing: 2) {
                     sideText
                         .createView(parentStyle: parentStyle.themeFont(fontSize: .smaller))
-
-                    Text("@")
-                        .themeFont(fontSize: .smaller)
-                        .themeColor(foreground: .textTertiary)
-
-                    Text(leverage ?? "")
+                    Text(size ?? "")
```

Comment:
if you have 20 positions open that are all leverage = 1, it is hard to use leverage as a risk factor since you have to add up each position's leverage

Given there is finite screen real estate, how might you suggest displaying leverage? I think the only helpful risk indicator would be total leverage. See screenshot


![Screenshot 2024-02-08 at 1 05 42 PM](https://github.com/dydxprotocol/v4-native-ios/assets/149746839/595b263a-484f-404c-9553-8733c190b338)

Reply:
@johnqh this change was reverted, please see updated screenshots for position (only change is coloring and inverting $/% for unrealized PnL